ferrocene/doc/user-manual/src/targets/x86_64-unknown-linux-gnu.rst

```diff
@@ -6,26 +6,28 @@
 x86_64 Linux (glibc)
 ====================
 
-The ``x86_64-unknown-linux-gnu`` Ferrocene target provides support for
-Linux on x86_64 using glibc.
+The ``x86_64-unknown-linux-gnu`` Ferrocene target provides support for Linux on
+x86_64 using glibc 2.27 or higher.
 
 Prerequisites
 -------------
 
-This target requires the ``x86_64-linux-gnu-gcc`` compiler, and is qualified
-for use with version 7.5.x, as shipped by Ubuntu 18.04 LTS.
+This target uses the LLVM ``ld.lld`` linker but to locate the system C libraries
+required to link a functional Linux binary, this target drives the ``ld.lld``
+linker using your system's C compiler as a linker-driver.
 
-On Ubuntu 18.04 LTS you can install it with:
+You must have a C compiler which:
 
-.. code-block::
-
-   $ sudo apt install gcc
+* Supports ``-fuse-ld=ld.lld`` option to select ``ld.lld`` as the linker
+* Supports ``-B`` option to modify the tool search path so it can find Ferrocene's
+  copy of ``ld.lld``
+* Does not load plugins into the linker (e.g. the GCC LTO plugin)
```

Comment:
I'd add a line "only passes the command line flags in [link to safety manual] to ld.lld".

Reply:
Added